master/reference/calicoctl/resources/policy.md

```diff
@@ -66,14 +66,19 @@ connection tracking is always used for flows to and from those.
 
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
-| action      | Action to perform when matching this rule. | allow, deny, log | string | |
+| action      | Action to perform when matching this rule. | allow, deny, log, pass | string | |
 | protocol    | Positive protocol match.  | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | notProtocol | Negative protocol match. | tcp, udp, icmp, icmpv6, sctp, udplite, integer 1-255. | string | |
 | icmp        | ICMP match criteria.     | | [ICMP](#icmp) | |
 | notICMP     | Negative match on ICMP. | | [ICMP](#icmp) | |
 | source      | Source match parameters. |  | [EntityRule](#entityrule) | |
 | destination | Destination match parameters. |  | [EntityRule](#entityrule) | |
 
+An `action` of `pass` will skip over the remaining Policies and jump to the
+Profile assigned to the endpoint, applying the policy configured in the
```

Comment:
There can be more than one profile referenced by an endpoint; they are executed in sequence.